app/controllers/products_controller.rb

```diff
@@ -22,6 +28,7 @@ def edit
   # POST /products or /products.json
   def create
     @product = Product.new(product_params)
+    @categories = Category.pluck(:name, :id)
```

Comment:
let's use Category.pluck(:name, :id) directly in the view so we don't have to set it in all new, create, edit and update

Reply:
Done